toolchest_client/api/streaming.py

```diff
@@ -0,0 +1,53 @@
+"""
+toolchest_client.api.streaming
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module provides a StreamingClient object, used by Toolchest queries to
+receive and print output lines streamed from the Toolchest server.
+"""
+import asyncio
+import ssl
+
+import websockets
+from websockets.exceptions import ConnectionClosed
+
+
+class StreamingClient:
+    """A Toolchest output stream client.
+
+    Provides an interface to output lines streamed from the server.
+
+    """
+
+    def __init__(self):
+        self.ssl_context = None
+        self.streaming_token = None
+        self.streaming_ip_address = None
+        self.streaming_tls_cert = None
+        self.username = "toolchest"
+        self.params_initialized = False
```

Comment:
`params_initialized` is a bit confusing of a name, is the meaning the same as `initialized`?

Reply:
Essentially, so I'll refactor it